OfekTwitter/css/users.css

```diff
@@ -1,4 +1,4 @@
-
+/* ok */
 body{
     font-family: Arial,Algerian;
 }
```

Comment:
you implemented the id `#paload` both here and in "index.html"
instead, you can implement it in a class and use it in both places
and place it in the general css file